esp/esp/program/modules/handlers/adminclass.py

```diff
@@ -388,11 +388,12 @@ def coteachers(self, request, tl, one, two, module, extra, prog):
                 error = 'Error - You already added this teacher as a coteacher!'
 
             if error:
-                return render_to_response(self.baseDir()+'coteachers.html', request,{'class':cls,
-                                                                                                 'ajax':ajax,
-                                                                                                 'txtTeachers': txtTeachers,
-                                                                                                 'coteachers':  coteachers,
-                                                                                                 'error': error})
+                return render_to_response(self.baseDir()+'coteachers.html', request,
+                                          {'class':cls,
```

Comment:
If you feel like it you could clean up the spaces after the colons too while you're at it.